src/main/java/edu/harvard/iq/dataverse/DatasetPage.java

```diff
@@ -3074,6 +3064,27 @@ public void updateFileCounts(){
         }
     }
     
+    
+    private Long originalSelectedFileSize;
+    
+    private Long archivalSelectedFileSize;
+
+    public Long getOriginalSelectedFileSize() {
+        return originalSelectedFileSize;
+    }
+
+    public void setOriginalSelectedFileSize(Long originalSelectedFileSize) {
+        this.originalSelectedFileSize = originalSelectedFileSize;
+    }
+
+    public Long getArchivalSelectedFileSize() {
+        return archivalSelectedFileSize;
+    }
+
+    public void setArchivalSelectedFileSize(Long archivalSelectedFileSize) {
+        this.archivalSelectedFileSize = archivalSelectedFileSize;
+    }
+
```

Comment:
are these used? I couldn't find their references.

Reply:
Removed. abandoned in favor of other/existing methods.